CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Deprecated
 
 ### Removed
+- Remove banner about bwc tests being disabled on mac ([#18983](https://github.com/opensearch-project/OpenSearch/pull/18983))
```

Comment:
Nitpick, but this commit message and changelog entry should be "Enable backward compatibility tests on Mac". You're changing how the tests function on a mac, not just removing the banner.